### PR TITLE
Navigate with keyboard

### PR DESCRIPTION
**Pull request by LetsBuild-exe**
Makes the contextmenu navigatable with keyboard, to faster select recipees

### CODE REVIEW

File: ficsit-companion/src/main.cpp [hunk 1]
```diff
@@ -113,8 +113,9 @@ bool Render(SDL_Window* window, App* app)
 
     ImGui::SetNextWindowPos(ImVec2(0, 0));
     ImGui::SetNextWindowSize(ImGui::GetIO().DisplaySize);
-
+
```
Comment (adepierre) [inline ficsit-companion/src/main.cpp]: Trailing whitespaces should be removed

**Review by adepierre — COMMENTED** [review]
Thanks for the PR. I won't be ablet to check it in details before next week to see if it impacts something else.

File: ficsit-companion/src/main.cpp (hunk 1 above)
Comment (LetsBuild-exe) [reply ficsit-companion/src/main.cpp]: ahh thought it would be to seperate, cause in 113 theres also a whitespace

Comment (adepierre) [reply ficsit-companion/src/main.cpp]: There is a new line, but not a new line with "useless" spaces

113 is ``\n``
116 is ``    \n``

Comment (LetsBuild-exe) [reply ficsit-companion/src/main.cpp]: Ah now i get it, how du i commit my change in the pull request?
or do i make a new one

Comment (adepierre) [reply ficsit-companion/src/main.cpp]: I think if you push the commit to your branch it will be added to the PR

Comment (LetsBuild-exe) [reply ficsit-companion/src/main.cpp]: fixed, thanks commited to wrong branch xD

**Pull request merged**